src/users/views.py

```diff
@@ -199,6 +199,13 @@ def get_context_data(self, **kwargs):
         return context
 
 def review_stages(request):
+    
+    review_stages_list = [
+        'Call for Proposals', 'First Round Review_1',
+        'First Round Review_2', 'Modification Stage',
```

Comment:
Can we name these something better than `First Round Review_1` and `First Round Review_2`?

Reply:
Refer to   [Proposal Review System Documentation](https://github.com/pycontw/pycon.tw/blob/master/src/reviews/README.md), @yychen Can we call `First Round Review_1` to `Filter problematic proposals` and `First Round Review_2` to `First Round Review` ?

Reply:
No problem with `First Round Review_2` to `First Round Review`. How about `Pre First Round Review`? Or since it is all locked, I think we can also call it something like `Locked (proposal editing and reviewing disabled)`?

Reply:
What @yychen suggested seems to be a better naming to me

Reply:
Thanks for the discussion
I push a new commit 88ee598f295eaae913aa647352883274e4bcb39d.